smrf/data/loadGrid.py

```diff
@@ -97,12 +87,15 @@ def model_domain_grid(self):
 
         dlat = np.zeros((2,))
         dlon = np.zeros_like(dlat)
-        dlat[0], dlon[0] = utm.to_latlon(np.min(self.x), np.min(self.y),
-                                         int(self.dataConfig['zone_number']),
-                                         self.dataConfig['zone_letter'])
-        dlat[1], dlon[1] = utm.to_latlon(np.max(self.x), np.max(self.y),
-                                         int(self.dataConfig['zone_number']),
-                                         self.dataConfig['zone_letter'])
+        dlat[0], dlon[0] = utm.to_latlon(np.min(self.topo.x),
+                                         np.min(self.topo.y),
+                                         self.topo.zone_number,
+                                         northern=self.topo.northern_hemisphere)
+
+        dlat[1], dlon[1] = utm.to_latlon(np.max(self.topo.x),
+                                         np.max(self.topo.y),
+                                         self.topo.zone_number,
+                                         northern=self.topo.northern_hemisphere)
```

Comment:
Correct me if I am wrong here, but the variable value assignments look identical to the one in the `init` call.
Can this be combined or variables re-used from the `init`?

Reply:
Looks like it. That was there before hand so I was mostly on a ctrl+f and replace mission. But I think it worth DRY-ing out while I am here.